src/NewCard/NewCard.stories.js

```diff
@@ -0,0 +1,197 @@
+      <Card>
+        <CardSection>This is content of card</CardSection>
+      </Card>
+    );
+  })
+  .add("Card with sections", () => {
+    return (
+      <Card>
+        <CardSection title={sectionTitle} description={sectionDescription} />
+        <CardSection title={sectionTitle} description={sectionDescription} />
+        <CardSection title={sectionTitle} description={sectionDescription} />
+      </Card>
+    );
+  })
+  .add(
+    "Card with expandable sections",
+    () => {
+      return (
+        <Card title={title} description={description}>
+          <>
+            <CardSection expandable title={sectionTitle}>
+              This is a card content
+            </CardSection>
+          </>
+          <div>
+            <CardSection expandable title={sectionTitle}>
+              This is a card content
+            </CardSection>
+          </div>
+          <CardSection expandable title={sectionTitle}>
+            This is a card content
+          </CardSection>
+        </Card>
+      );
+    },
+    {
+      info:
+        "Card sections allow you to create separate sections in every card when you need to create more advanced content structure. Visit Orbit.Kiwi for more detailed guidelines.",
+    },
+  )
+  .add(
+    "Card with default expanded",
+    () => {
+      const initialExpanded = boolean("initialExpended", true);
+
+      return (
+        <Card>
+          <CardSection
+            expandable
+            icon={<CarrierLogo size="large" carriers={[{ code: "FR", name: "Ryanair" }]} />}
+            title={
+              <Stack align="center" flex justify="between">
+                <Stack align="center">
+                  <Text type="secondary">Trip length: 1h55m</Text>
+                  <Badge icon={<Clock />} type="warning">
+                    Unavailable
+                  </Badge>
+                </Stack>
+              </Stack>
+            }
+          >
+            Hidden content
+          </CardSection>
+          <CardSection
+            expandable
+            expanded={initialExpanded}
+            onExpand={action("onExpand")}
+            icon={<CarrierLogo size="large" carriers={[{ code: "FR", name: "Ryanair" }]} />}
+            onClose={action("onClose")}
+            title={<Text type="secondary">Trip length: 1h55m</Text>}
+          >
+            By default visible content
+          </CardSection>
+        </Card>
+      );
+    },
+    {
+      info:
+        "Card sections allow you to create separate sections in every card when you need to create more advanced content structure. Visit Orbit.Kiwi for more detailed guidelines.",
+    },
+  )
+  .add(
+    "Card with mixed sections",
+    () => {
+      return (
+        <Card title={title} description={description}>
+          <CardSection title={sectionTitle} description={sectionDescription}>
+            Content
+          </CardSection>
+          <CardSection expandable title={sectionTitle} description={sectionDescription}>
```

Comment:
The arrow icon should be aligned to the center of title + description.
<img width="1195" alt="Snímek obrazovky 2019-11-26 v 13 52 37" src="https://user-images.githubusercontent.com/7254437/69635206-345c8f80-1054-11ea-9c4a-5651d799eabb.png">